build.sh

```diff
@@ -42,6 +42,7 @@ install_deps () {
     go get github.com/jgautheron/goconst/cmd/goconst
     go get honnef.co/go/tools/cmd/megacheck
     go get github.com/golang/lint/golint
+    install_golang_dep
```

Comment:
Just noticed that `install_golang_dep` does `go get` on dep. But that gives an unspecified version of dep (master).

We should probably think about _not_ doing that, because it makes our builds not reproducible if people are using different versions of dep.

Reply:
Is there another way we can install our vendor dependencies in CI at least? Because it requires `go get` on dep if we want to `dep ensure` right?

Reply:
I'll wait for @echlebek PR to get merged that will install our dependencies differently for CI and remove `install_golang_dep` from build.sh and build.ps1 so our builds will be reproducible. I'll rebase from master once that's done 😅